process/project-lifecycle.md

```diff
@@ -0,0 +1,178 @@
+* Maintain a diversified contributor base (i.e. not a single-vendor project) with an active flow of contributions
+* Follow security best practices (as recommended by the OpenSSF and others)
+* Maintain a point of contact for vulnerability reports
+* Implement, practice, and refine mature software development and release practices such as following a version schema
+* Begin to establish the appropriate governance that enables its sustainment for potential graduation
+
+#### Project Support
+* Receives guidance on technical direction from TAC
+* Receives a TAC or WG sponsor for guidance as an Incubation project
+* Receives space for Contributor Summits at OpenSSF bi-annual events
+* Receives OpenSSF Code of Conduct Committee support
+* Receives infrastructure support (details determined by project leads and OpenSSF Budget Committee)
+* Receives support with vulnerability disclosure from the OpenSSF (Vulnerability Disclosure WG)
+* May post project updates and tutorials to the OpenSSF blog
+* May request OpenSSF budget for project improvements such as security audits or time-bound contracting needs
+* With additional TAC or WG approval, may fundraise for dedicated project funds, coordinated by the OpenSSF
+* Projects may use the OpenSSF logo to promote their project (in accordance with the trademark guidelines). Projects may not be referred to as an "OpenSSF Project" or "OpenSSF $ProjectName." Projects may say they are an "OpenSSF Incubating Project."
+
+#### Incubation Entry Requirements and Considerations
+
+* Projects must have a minimum of three maintainers with a minimum of two different organization affiliations.
+* Projects should be able to show adoption by multiple parties and adoption's value to the open source community and/or end users (may include adoption of beta/early versions).
+* Projects must be aligned with the OpenSSF mission and either be a novel approach for existing areas, address an unfulfilled need, or be initial code needed for OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
+* Projects must have documented, initial project governance.
+
+#### Project Process: Sandbox to Incubation and direct entry to Incubation
+
+Projects are required to undergo technical due diligence as a part of the process of entering Incubation, whether as a move from Sandbox to Incubation or enterting Incubation directly. Technical Due Diligence is driven by a TAC or parent WG sponsor with close collaboration of the project. Once the diligence is complete and the proposal made, the Due Diligence document is made available to the community for two weeks to solicit public comment before a TAC vote is called.
+
+
+
+### Graduated
+
+Graduated projects signal the highest level of maturity for an OpenSSF project. Graduated projects have consistent, mature open source development practices with sustained contribution and activity. 
```

Comment:
Do we want to specify projects also have users (we mention end users in other places). While I think it's assumed I think we should be explicit

Reply:
This is covered on line 122